Makefile

```diff
@@ -69,18 +78,38 @@ fix: ## Fix linting issues in the code
 	${DOCKER_COMPOSE} run --rm app npm run lint:fix
 
 test: export TARGET = dev
-test: ## Run the tests
+test: ## Run all the tests
+	$(MAKE) db
+	$(MAKE) wait-healthy
+	$(MAKE) initdb
 	${DOCKER_COMPOSE} run --rm app npm run test
 
+unit-test: export TARGET = dev
+unit-test: ## Run the unit tests
+	${DOCKER_COMPOSE} run --rm app npm run test:unit
+
+integration-test: export TARGET = dev
+integration-test: ## Run the integration tests
+	$(MAKE) db
+	$(MAKE) wait-healthy
+	$(MAKE) initdb
+	${DOCKER_COMPOSE} run --rm app npm run test:integration
```

Comment:
This will leave `db` running won't it?

Reply:
Yes; can be unexpected. Catching errors, stopping it and raising it again seems an headache though.

Reply:
https://github.com/libero/article-store/blob/9f9e78331ae2c0e90d64d080b4c58667e44b19dc/Makefile#L76 works successfully.